example/ios/Demo/KeyboardManager/KeyboardManagerViewController.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  KeyboardManagerViewController.swift
+//  Demo
+//
+//  Created by Gijs van Veen on 04/12/2019.
+//  Copyright © 2019 Splendo. All rights reserved.
+//
+
+import UIKit
+import KotlinNativeFramework
+
+class KeyboardManagerViewController : UIViewController {
+    
+    @IBOutlet private var editField: UITextField!
+    private var exampleKeyboardManager: SharedExampleKeyboardManager!
```

Comment:
Actually force unwrapped variables is not good practice (exclude @IBOutlet).

Probably `let exampleKeyboardManager = KotlinNativeFramework().keyboardManager(textField: editField)` should works, if not then better:
`lazy var exampleKeyboardManager = KotlinNativeFramework().keyboardManager(textField: editField)`

Reply:
I disagree. First of all, these are not forced unwraps but implicit unwraps. They are like that because on the init they cant be expected to be declared, but the developer can assume they are set whenever called. Since IBOutlet binding does not happen on initalization, setting the keyboardManager on itit will fail (as textField is nil).

Lazy initialization is also incorrect, because that would mean its not initialized if not called, OR wrongly initialized if called before IBOutlet binding occurs.

Instead, we should use implicit unwraps to signify that values must be set, but after that can be assumed to be non-nil.

Reply:
:+1:

Reply:
Agreed! But if you would write it assessment Hsiao would reject it )))